C4SGWeb/config/webpack.dev.js

```diff
@@ -0,0 +1,24 @@
+var webpackMerge = require('webpack-merge');
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
+var commonConfig = require('./webpack.common.js');
+var helpers = require('./helpers');
+
+module.exports = webpackMerge(commonConfig, {
+  devtool: 'cheap-module-eval-source-map',
+
+  output: {
+    path: helpers.root('dist'),
+    publicPath: 'http://localhost:8000/',
```

Comment:
No reason.  Moved back to 3000 and eliminated duplicate use of port by keeping it all in webpack config